app/client/styles/main.styl

```diff
@@ -48,6 +48,7 @@ html, body
   background-position: center
   min-height: 100%
   color: $color-black
+  visibility: hidden
```

Comment:
Ah rite I was seeing that recently. Can you include descriptions in PRs or split them in cases like this.